### DISCUSSION TITLE
Machine Learning Systems - 2  ML Systems

### DISCUSSION THREAD

**Discussion by giscus** [harvard-edge/cs249r_book#291] [Chapter Discussions]
# Machine Learning Systems - 2  ML Systems

https://harvard-edge.github.io/cs249r_book/contents/ml_systems/ml_systems.html

**Comment by clv07** [comment]
I don't think the links in the Resources section work.

**Reply by profvjreddi** [reply]
By the way, this is the dev version, which is down sometimes. Are you able to access the [main](https://mlsysbook.ai) version directly?

**Comment by profvjreddi** [comment]
Thank you for letting me know @clv07 -- I am going to push a big update soon, so stay tuned for that. I will make sure we fix it in that release.

**Reply by bhupesh-sf** [reply]
I also visited just today and found few links leading to 404
